### DISCUSSION TITLE
780

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#952] [Comments]
# 780

학생 시절에 좋아하던 이성이 있었다. 좋아하니깐 더 잘해주고 싶었고, 내가 하는 행동의 순위에서 항상 최우선이였다. 근데 고백을 하면 계속 거절했다. 진심과는 별개로 인연은 아니였다. 친구로서 계속 지내다가 어느날 그 친구에게서 아프다는 연락을 받았다. 보통 같았으면 연락을 받자마자 걱정하고 달려갔을텐데, 그날은 그 연락을 받고 "근데 내가 약 들고 간다고 사귀게 되는 것도 아니잖아? 그럼 의미가 있나?" 라는 생각이 들었다. 이런 생각을 했다는 것에 소스라치게 놀랐다. 그 친구가 좋아서 해온 모든 행동을 나 스스로 의미가 없는 행동으로 만들어버린 것이기 때문이다. 좋아서 무엇이든 해주고 싶었던 순수한 진심이 오염된 것 같았다. 사귀는게 확실할때만 내가 해온 노력이 모두 의미가 있는 것이고, 사귀지 않을..

https://jojoldu.tistory.com/780

**Comment by sang5c** [comment]
굳이 이런 일을 했던 분들이 팀장이 되고,
열심히 하는 사람들의 연봉이 올라가고,
이렇게 하는 사람들이 10억 부자가 됐으면 좋겠네요. ㅎㅎ

좋은 글 감사합니다. 잘 보고 있습니다 :)

**Reply by jojoldu** [reply]
저도 동의합니다 :) 
그런 분들이 종국엔 더 잘되기를 바래봅니다 :)

**Comment by jws1837** [comment]
공감합니디. 우리가 통제할 수 있는 것은 과정뿐이고, 그것에 의미부여를 하는 것 또한 나 자신이기에 주체성을 가지려면 과정에 집중할 수밖에 없는 것 같습니다.

**Comment by jiwooproity** [comment]
의미에 매몰되어 좋아했던 일이 변질되는 일만큼 안타까운 것이 없을 거 같습니다.
내가 선택한 길, 내가 선택한 일인 만큼 행복함을 느끼는 지금을 더욱 단단하게 만들어야겠습니다.
스스로를 돌아보며 나도 의미에 매몰된 적은 없는가를 생각해 보며, 다시 한번 생각을 정리해 보는 계기가 되었습니다.
좋은 글 감사합니다.
